src/EnergyPlus/WeatherManager.cc

```diff
@@ -3731,6 +3750,27 @@ namespace WeatherManager {
         return (fmod(interpAng, 360.)); // fmod is float modulus function
     }
 
+    Real64 CalcSkyEmissivity(int ESkyCalcType, Real64 OSky, Real64 DryBulb, Real64 DewPoint, Real64 RelHum){
```

Comment:
Why have `RelHum` as an argument when `DryBulb` and `DewPoint` are also here?

Reply:
RelHum is used to calculate the partial water pressure directly. Otherwise, kinda have to convert it inside function.

Reply:
> @xuanluo113 you'll need to address the crash and the question from @mjwitte about the function signature.
> 
> `CalcSkyEmissivity` is declared as:
> 
> ```
> Real64 CalcSkyEmissivity(..., Real64 DryBulb, Real64 DewPoint, Real64 RelHum);
> ```
> 
> With relative humidity as the last argument. You also have dry bulb and dew point so unless you are saying the barometric pressure is changing, you only need to pass in two psychrometric values to locate your state. However, even with that variable there, you are calling it with humidity ratio:
> 
> ```
> ESky = CalcSkyEmissivity(..., DryBulb, TomorrowOutDewPointTemp(TS, Hour), OutHumRat);
> ```
> 
> And that's causing a crash because `OutHumRat` is uninitialized at that point.
> 
> Please address that as soon as possible and make sure to pull develop in right before you push your changes so we get clean CI results.

@Myoldmopar @mjwitte Yes. I'm working on it now. But the calculation of partial vapor pressure requires relative humidity directly, while the other places requires dew point. If I only pass two psychrometric vars, I have to re-do the psychrometric conversion again inside function, which is a kind of duplicate calculation?